CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.5.0 (2018-04-16)
+--------------------------
+
+- Adds new endpoint to compare two workflows. The output is a ``git`` like
+  diff which can be configured to show differences at metadata level,
+  workspace level or both.
+- Adds new endpoint to retrieve workflow parameters.
+- Adds new endpoint to query the disk usage of a given workflow.
+- Adds new endpoints to move and delete files in workflow workspace.
+- Adds a new endpoints to open and close interactive sessions inside the
```

Comment:
If you agree with 1st comment, it could be just ``an interactive session inside the workspace.``